timesync/manual_clock.go

```diff
@@ -0,0 +1,28 @@
+package timesync
```

Comment:
I'd move this implementation to same as the real clock, since these two struct need to implement same interface... looks kinda Javrad ;)

Reply:
actually, it may be best if this moved into `multi_node_sim`, the only place it's actually used

Reply:
I agree. Moved.